src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "nvidia-cuda",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
We might want to bump the major version, since this would break compatibility with Ubuntu 20.04 devcontainers. @eljog Do you agree?

Reply:
Same here, major version bump should be 2.0.0